Sources/Mantis/CropViewConfig.swift

```diff
@@ -0,0 +1,20 @@
+import UIKit
+
+public struct CropViewConfig {
+    public var cropViewMinimumBoxSize: CGFloat = 42
+    public var cropViewPadding: CGFloat = 14
+    public var hotAreaUnit: CGFloat = 32
```

Comment:
Should we document, what exactly this is? I have no idea what it is. Looks good though.

Reply:
<img width="784" alt="image" src="https://user-images.githubusercontent.com/26723384/179739837-41178283-6e69-4f6d-aa1d-526e07a0f989.png">
It is for how easily you can drag GridOverlayView. It was used internally so I didn't document it. It probably needs a better name now.

Reply:
Ah tough thing to find a name for.

Reply:
Rename hotAreaUnit to cropBoxHotAreaUnit